src/main/frontend/components/sidebar.css

```diff
@@ -48,6 +46,7 @@
   position: relative;
   height: 100%;
   transition: padding-left .3s;
+  user-select: none;
```

Comment:
This will disable user-select to avoid conflicting situations like this
![Screenshot from 2023-03-28 20-33-15](https://user-images.githubusercontent.com/10744960/228321562-e694cf77-0c6f-448e-a874-70574497a83b.png)
but we might have to revert this if there is a reason to highlight text on select mode.